cst438_gradebook/src/main/java/com/cst438/controllers/EnrollmentController.java

```diff
@@ -17,24 +17,26 @@
 @RestController
 public class EnrollmentController {
 
-	@Autowired
-	CourseRepository courseRepository;
-
-	@Autowired
-	EnrollmentRepository enrollmentRepository;
-
```

Comment:
Ryan was able to add some new lines to help out with the appearance and option of of the student name and email. This was a good addition to handle the appropriate execution of the studentName and studentEmail